assets/js/popup.js

```
@@ -66,64 +68,72 @@ async function handleTranscriptAction(format, mode) {
     // CSV Header
-    content = "Timestamp,Speaker,Transcript\n";
-    content += data.map(item => {
-      // Escape quotes by doubling them, and wrap fields in quotes
-      const escape = (text) => `"${(text || "").replace(/"/g, '""')}"`;
-      return `${escape(item.timestamp)},${escape(item.speaker)},${escape(item.text)}`;
-    }).join("\n");
+    content = 'Timestamp,Speaker,Transcript\n';
+    content += data
+      .map((item) => {
+        // Escape quotes by doubling them, and wrap fields in quotes
+        const escape = (text) => `"${(text || '').replace(/"/g, '""')}"`;
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)